packages/client/base/src/types/hosted/v3/CrossmintHostedCheckoutV3Props.ts

```diff
@@ -0,0 +1,26 @@
+import type { Locale } from "@/types";
+import type { EmbeddedCheckoutV3LineItem, EmbeddedCheckoutV3Payment, EmbeddedCheckoutV3Recipient } from "@/types/embed";
+
+export interface CrossmintHostedCheckoutV3Props {
```

Comment:
Have this interface been reviewed?

Reply:
yeah its in the v3 doc 

theres really not much to review tho, its 95% the same as embed - only diff is appearance object, and payment doesnt take a `payer` signer object

![image](https://github.com/user-attachments/assets/20e82c32-3812-4fa3-a686-28ff36ab195e)

https://docs.google.com/document/d/1y602C9MwV8HZi9aLTDR0KANOcvnTZEP0tvqmE_OqXDA/edit?tab=t.0